media-gfx/gscan2pdf/gscan2pdf-2.5.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils perl-module
+
+DESCRIPTION="Scan documents, perform OCR, produce PDFs and DjVus"
+HOMEPAGE="http://gscan2pdf.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3"
```

Comment:
Please verify the license.

https://devmanual.gentoo.org/general-concepts/licenses/index.html

Reply:
We have:
https://sourceforge.net/p/gscan2pdf/code/ci/v2.5.3/tree/LICENCE
[a55f78] Jeffrey Ratcliffe updated LICENCE file to GPLv3 | 2007-10-12 14:36:16

But gscan2pdf.appdata.xml has:
<metadata_license>CC0-1.0</metadata_license>
<project_license>GPL-3.0+</project_license>

and files:
icons/gscan2pdf.svg
icons/scanner.svg
have:
        <cc:license
           rdf:resource="http://creativecommons.org/licenses/GPL/2.0/" />

I can ask about the two icons to see if it can be simplified at least to two.

Reply:
LICENCE file doesn't matter.

Reply:
There's nothing in the code files. There is a COPYING that says GPL 3 (no or later) and:
> The scanner icon was taken from
http://quantum-bits.org/wp-content/uploads/2007/05/epson-perfection-1260.tar.gz
and is equally GPL.
>
> The PDF icon was taken from
http://upload.wikimedia.org/wikipedia/commons/b/b6/Gnome-mime-application-pdf.svg
and is LGPL.
>
> I created the gscan2pdf icon by combining the above two icons.

Reply:
Looks good. Sorry for the passive-aggressive tone.

Reply:
Thanks for the apology, no worries though.